DSharpPlus/Entities/Channel/DiscordDmChannel.cs

```diff
@@ -81,11 +81,10 @@ public Task AddDmRecipientAsync(ulong user_id, string accesstoken, string nickna
         /// Only use for Group DMs!
         /// </summary>
         /// <param name="user_id">The ID of the User to remove.</param>
-        /// <param name="accesstoken">The OAuth2 access token.</param>
         /// <exception cref="Exceptions.NotFoundException">Thrown when the channel does not exist.</exception>
         /// <exception cref="Exceptions.BadRequestException">Thrown when an invalid parameter was provided.</exception>
         /// <exception cref="Exceptions.ServerErrorException">Thrown when Discord is unable to process the request.</exception>
-        public Task RemoveDmRecipientAsync(ulong user_id, string accesstoken)
```

Comment:
why was this removed?

Reply:
access token had no reference to its method in DiscordApiClient I preferred to delete it

Reply:
that's an unrelated issue to this PR, and we should consider it separately.